fdbserver/storageserver.actor.cpp

```diff
@@ -2491,6 +2489,7 @@ ACTOR Future<std::pair<ChangeFeedStreamReply, bool>> getChangeFeedMutations(Stor
 	state int remainingLimitBytes = CLIENT_KNOBS->REPLY_BYTE_LIMIT;
 	state int remainingDurableBytes = CLIENT_KNOBS->REPLY_BYTE_LIMIT;
 	state Version startVersion = data->version.get();
+	state ReadOptions options;
```

Comment:
Good addition here.  I'm not sure what the options should be, but let's add a TODO for someone to set them later.
```suggestion
	// TODO: Change feed reads should probably at least set cacheResult to false, possibly set a different ReadType as well, perhaps high priority?
	state ReadOptions options;
```